vendor/github.com/stretchr/testify/assert/yaml/yaml_custom.go

```diff
@@ -1,5 +1,4 @@
 //go:build testify_yaml_custom && !testify_yaml_fail && !testify_yaml_default
-// +build testify_yaml_custom,!testify_yaml_fail,!testify_yaml_default
```

Comment:
Interesting, but guess it doesn't do much for the `go.mod` dependencies, as that would still see it. ( see the comment below about "not linking with ..")